### PR TITLE
Ask VA/1129 - School search

### PR DESCRIPTION
**Pull request by ediiotero**
## Are you removing, renaming or moving a folder in this PR?
- [X] No, I'm not changing any folders (skip to Summary and delete the rest of this section)
- [ ] Yes, I'm removing, renaming or moving a folder

Did you change site-wide styles, platform utilities or other infrastructure?
- [X] No
- [ ] Yes, and I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#that-sounds-normal-so-whats-the-proxy-all-about) to test the injected header scenario

## Summary

- School search by code and fix for user prefill data
- _(If bug, how to reproduce)_
- _(What is the solution, why is this the solution)_
- Ask VA team owns this form
- _(If using a flipper, what is the end date of the flipper being required/success criteria being targeted)_

## Related issue(s)

- Ticket [1129](https://app.zenhub.com/workspaces/ask-va-647a476551689d06655cc815/issues/gh/department-of-veterans-affairs/ask-va/1129), [1130](https://app.zenhub.com/workspaces/ask-va-647a476551689d06655cc815/issues/gh/department-of-veterans-affairs/ask-va/1130)

## Testing done

- Search by school code
- Search by school code ex: 11904106
- Manual testing
- _Exclusively stating 'Specs and automated tests passing' is NOT acceptable as appropriate testing
- _Optionally, provide a link to your [test plan](https://depo-platform-documentation.scrollhelp.site/developer-docs/create-a-test-plan-in-testrail) and [test execution records](https://depo-platform-documentation.scrollhelp.site/developer-docs/execute-tests-in-testrail)_

## Screenshots

Screenshot 2024-08-29 at 8 44 50 AM
Screenshot 2024-08-29 at 8 42 08 AM

## What areas of the site does it impact?

Only impacts the Ask VA form

### Quality Assurance & Testing

- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [X] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [X] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [X] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [X] Did you login to a local build and verify all authenticated routes work as expected with a test user

### CODE REVIEW

File: src/applications/ask-va/components/search/EducationSearchItem.jsx [hunk 1]
```diff
@@ -62,6 +76,14 @@ const EducationSearchItem = ({
               uswds
```
Comment (hemeshvpatel) [inline src/applications/ask-va/components/search/EducationSearchItem.jsx:76]: Don't need the uswds anymore

File: src/applications/ask-va/components/search/EducationSearchItem.jsx [hunk 2]
```diff
@@ -62,6 +76,14 @@ const EducationSearchItem = ({
...
             />
           ))}
+          <va-radio-option
+            id="facility-not-listed"
+            label="My facility is not listed"
+            value="My facility is not listed"
+            name="primary"
+            checked={selected === 'My facility is not listed'}
+            uswds
```
Comment (hemeshvpatel) [inline src/applications/ask-va/components/search/EducationSearchItem.jsx:85]: remove uswds

File: src/applications/ask-va/components/search/EducationSearchItem.jsx (hunk 1 above)
Comment (ediiotero) [reply src/applications/ask-va/components/search/EducationSearchItem.jsx:76]: done

File: src/applications/ask-va/components/search/EducationSearchItem.jsx (hunk 2 above)
Comment (ediiotero) [reply src/applications/ask-va/components/search/EducationSearchItem.jsx]: done

**Review by hemeshvpatel — DISMISSED** [review]
Lgtm!

**Review by joehall-tw — APPROVED** [review]
LGTM!

**Review by rmessina1010 — APPROVED** [review]
LGTM

**Pull request merged**